esp/templates/users/userchecklist.html

```diff
@@ -23,10 +38,16 @@ <h1>Select users</h1>
 <form action="{{ nextpage }}" method="post" name="checklist">
 
 <table>
+<tr>
+<td>
+<input type="checkbox" id="chk_new" onclick="checkAll('chk');">
+Select all users
+</tr>
+</td>
 {% for user in users %}
 <tr>
 <td>
-<input type="checkbox" name="userno{{user.id}}" value="{{user.id}}" />
+<input type="checkbox" id="chk" name="userno{{user.id}}" value="{{user.id}}" />
```

Comment:
Since this is inside a `for`, there will be a bunch of them, so we should use `class` instead of `id`, since having multiple of the same `id` can confuse browsers.